### DISCUSSION TITLE
What's unique about this Plugin?

### DISCUSSION THREAD

**Discussion by Rumperuu** [markcheret/footnotes#113] [General]
There are at least ten bespoke footnoting Plugins available on the [WordPress directory](https://wordpress.org/plugins/search/footnotes/), and I expect that there are probably other Plugins out there that include footnoting amongst other features.

This Plugin is has the second-highest installbase of these Plugins, with 6,000+ to [Easy Footnotes](https://wordpress.org/plugins/easy-footnotes/)' 10,000+ (Easy Footnotes was last updated 8 months ago and is not yet certified compatible with WP 5.7, but other than that I haven't played around with it so I can't speak to its features, performance, etc.).

So, what is the USP of this Plugin in particular? Why should someone choose it over one of the many alternatives? This is important to decide on as a project, as it will help us to prioritise development, etc. going forward.

Do we want to be the most reliable, or the most customisable, or the most lightweight, or something else entirely?

**Comment by markcheret** [comment]
I'd say
- most customisable. Others I found at the time forced you to use a specific way of denoting footnotes. We ship with a default but ultimately leave it up to the user.
- other customisations already present, more to come
- this plugin used to follow offline publishing guidelines and was originally intended for the scientific community wanting to publish their papers online, keeping with an existing workflow and the way used to expressing themselves.
